my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency-phone.html

```diff
@@ -0,0 +1,41 @@
+<loading-gif data-object='emergencyPhoneNumbers' data-reuse="true"></loading-gif>
+<div class='info-card' ng-if="empty || emergencyPhoneNumbers.emergencyPhoneNumbers.length>0">
+  <h1>My Emergency Phone Number</h1>
+  <div class="description-box">
+    <p>An emergency phone number where you can be reached in cases of safety</p>
```

Comment:
"in cases of safety" doesn't speak to me.  Maybe something more like

"A phone number the university should call when attempting to contact you in an emergency."